tests/components/zha/test_select.py

```diff
@@ -413,3 +420,80 @@ async def test_on_off_select_attribute_report(
         hass, cluster, {"motion_sensitivity": AqaraMotionSensitivities.Low}
     )
     assert hass.states.get(entity_id).state == AqaraMotionSensitivities.Low.name
+
+
+(
+    add_to_registry_v2("Fake_Manufacturer", "Fake_Model")
+    .replaces(MotionSensitivityQuirk.OppleCluster)
+    .enum(
+        "motion_sensitivity",
+        AqaraMotionSensitivities,
+        MotionSensitivityQuirk.OppleCluster.cluster_id,
+    )
+    .enum(
+        "motion_sensitivity_disabled",
+        AqaraMotionSensitivities,
+        MotionSensitivityQuirk.OppleCluster.cluster_id,
+        translation_key="motion_sensitivity_translation_key",
+        initially_disabled=True,
+    )
+)
+
+
+@pytest.fixture
+async def zigpy_device_aqara_sensor_v2(
+    hass: HomeAssistant, zigpy_device_mock, zha_device_joined_restored
+):
+    """Device tracker zigpy Aqara motion sensor device."""
+
+    zigpy_device = zigpy_device_mock(
+        {
+            1: {
+                SIG_EP_INPUT: [
+                    general.Basic.cluster_id,
+                    MotionSensitivityQuirk.OppleCluster.cluster_id,
+                ],
+                SIG_EP_OUTPUT: [],
+                SIG_EP_TYPE: zha.DeviceType.OCCUPANCY_SENSOR,
+            }
+        },
+        manufacturer="Fake_Manufacturer",
+        model="Fake_Model",
+    )
+
+    zha_device = await zha_device_joined_restored(zigpy_device)
+    return zha_device, zigpy_device.endpoints[1].opple_cluster
+
+
+async def test_on_off_select_attribute_report_v2(
+    hass: HomeAssistant, zigpy_device_aqara_sensor_v2
+) -> None:
+    """Test ZHA attribute report parsing for select platform."""
+
+    zha_device, cluster = zigpy_device_aqara_sensor_v2
+    assert isinstance(zha_device.device, CustomDeviceV2)
+    entity_id = find_entity_id(
+        Platform.SELECT, zha_device, hass, qualifier="motion_sensitivity"
+    )
+    assert entity_id is not None
+
+    # allow traffic to flow through the gateway and device
+    await async_enable_traffic(hass, [zha_device])
+
+    # test that the state is in default medium state
+    assert hass.states.get(entity_id).state == AqaraMotionSensitivities.Medium.name
+
+    # send attribute report from device
+    await send_attributes_report(
+        hass, cluster, {"motion_sensitivity": AqaraMotionSensitivities.Low}
+    )
+    assert hass.states.get(entity_id).state == AqaraMotionSensitivities.Low.name
+
+    entity_registry = er.async_get(hass)
+    entity_entry = entity_registry.async_get(
+        "select.fake_manufacturer_fake_model_none"  # none in id because the translation key does not exist
+    )
```

Comment:
Technically, the line is also a bit long. Maybe put the comment above and make it slightly more clear?
```suggestion
    # 'none' suffix in entity id, as the translation key used here does not exist
    entity_entry = entity_registry.async_get("select.fake_manufacturer_fake_model_none")
```